homeassistant/components/roborock/button.py

```diff
@@ -97,3 +120,28 @@ def __init__(
     async def async_press(self) -> None:
         """Press the button."""
         await self.send(self.entity_description.command, self.entity_description.param)
+
+
+class RoborockSceneButtonEntity(RoborockEntity, ButtonEntity):
```

Comment:
Can we call these routines?  It is ok ok if the initial variable above has scene in the name  and called a `scene_id` given that is how the python api returns it, but according to the user that filed the documentation they are called [Routines](https://support.roborock.com/hc/en-us/articles/7120664101785-What-can-I-do-with-the-Routines)

So we can leave the python data objects alone, but use `get_routines` and `execute_routine` in home assistant.

Reply:
done